README.md

```diff
@@ -17,6 +17,18 @@ There is an alternative implementation under [goto-opensource/secure-frame-ts](h
 * ratcheting is not implemented
 * keyIds are used as senderIds
 
+## Supported crypto libraries
+Currently two crypto libraries are supported:
+- [ring](https://crates.io/crates/ring) 
+    - is enabled per default with the feature `ring`
+    - supports compilation to Wasm32
+- [openssl](https://crates.io/crates/openssl)
+    - is enabled with the feature `openssl`
+        - To build e.g. use `cargo build --features openssl --no-default-features`
+    - uses rust bindings to `libopenssl`.
+    - Per default the library has to be manually installed on your system and is dynamically linked. For further options see the [openssl crate documentation](https://docs.rs/openssl/0.10.55/openssl/). 
+    - Compilation to Wasm32 is not supported
+
```

Comment:
does it not use https://github.com/alexcrichton/openssl-src-rs yet?

Reply:
Thanks :) Seems like one can enable it with a feature (`vendored`). I will play around with that & might enable it by default.